### DISCUSSION TITLE
Calling system

### DISCUSSION THREAD

**Discussion by mealicecooper** [itsfuad/Poketab-Messenger#49] [Q&A] [answered]
Can you implement audio and video calls here? Many webapp have this.

**Comment by itsfuad** [comment, marked as answer]
I'm working on it. It's very difficult to manage multiple media streams on the web browser without a relay server. Each client have to recieve multple streams and also have to send own stream to the other clients. So it will be a mesh network.

We are currently working on audio only system. Hope this will be implemented soon. 

Thanks!
